### PR TITLE
[vscode][3/n] Setup Env Variables: Use env template file for new files

### PR DESCRIPTION
**Pull request by rossdanlm**
[vscode][3/n] Setup Env Variables: Use env template file for new files

If the file does not exist, created a template file with some helper text to help guide people on how to set their environment variables.

Next PR I will handle the use case for when the env file already exists

## Test Plan

https://github.com/lastmile-ai/aiconfig/assets/151060367/4608833c-d64f-4906-a7ed-94af12174c6d

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1283).
* #1284
* __->__ #1283
* #1280

### CODE REVIEW

File: vscode-extension/static/env_template.env [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Uncomment the following lines to set these common env variables for
```
Comment (rossdanlm) [inline vscode-extension/static/env_template.env:1]: Tanya said this is too much text. I'm just going to land and we can fix later

File: vscode-extension/src/extension.ts [hunk 2]
```diff
@@ -756,53 +758,42 @@ async function setupEnvironmentVariables() {
...
+        preview: false,
+        // Tried using vscode.ViewColumn.Active but that overrides existing
+        // walkthrough window
+        viewColumn: vscode.ViewColumn.Beside,
+      });
+      vscode.window.showInformationMessage(
+        "Please define your environment variables."
+      );
+    }
+  }
```
Comment (Ankush-lastmile) [inline vscode-extension/src/extension.ts:796]: nit: what if it doesn't open?

**Review by Ankush-lastmile — APPROVED** [review]
lgtm, approving to unblock

File: vscode-extension/src/extension.ts (hunk 2 above)
Comment (rossdanlm) [reply vscode-extension/src/extension.ts:796]: Good catch, will follow up someday

Comment (rossdanlm) [reply vscode-extension/src/extension.ts:796]: https://github.com/lastmile-ai/aiconfig/issues/1287

**Pull request merged**